doc/cell.md

```diff
@@ -104,6 +104,21 @@ $ ni n4 ,e
 54.5981500331442
 ```
 
+These functions also `,e` and `,l` also take an optional paramter as a base:
+
+```bash
+$ ni n4 ,l2
+0
+0.999999999999998
```

Comment:
heh - this might be a problem :)

there are cases where the results are hardware/library-dependent -- i recommend quantizing each number to 0.001 or something before you assert anything about them: `ni n4 ,l2 ,q.001` maybe